index.js

```diff
@@ -215,6 +215,7 @@ export const Rels = {
 		questionResponse: 'https://quizzes.api.brightspace.com/rels/question-response',
 		quiz: 'https://quizzes.api.brightspace.com/rels/quiz',
 		section: 'https://quizzes.api.brightspace.com/rels/quiz-section',
+		studySupport: 'https://quizzes.api.brightspace.com/rels/studyRecommendations',
```

Comment:
Or should it be?
```suggestion
		studyRecommendations: 'https://quizzes.api.brightspace.com/rels/studyRecommendations',
```

Reply:
Or should it all be changes to be `studySupport` 🤔

Reply:
I wouldn't change the actual rel because it's already used in other places. But you can call it `studySupport` or `studyRecommendations`, I don't have strong opinions on that.